### PR TITLE
Add logs entries

### PR DESCRIPTION
**Pull request by Thomasfds**
It's my first fork and i don't understand all for this time lol.

But i have add somes logs endpoints for documentation.

### CODE REVIEW

File: docs/en-US/api/v1/endpoints/logs.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+# Player Activity
```
Comment (pandinocoder) [inline docs/en-US/api/v1/endpoints/logs.md]: Should be
```md
# Logs

## Player Activity
```

File: docs/en-US/api/v1/endpoints/logs.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+## Authorized
```
Comment (pandinocoder) [inline docs/en-US/api/v1/endpoints/logs.md]: Delete

File: docs/en-US/api/v1/endpoints/logs.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    ...
+  ],
+  "Sort": null,
+  "Extra": null
+}
+```
+
+# User Activity
+
+## Authorized
```
Comment (pandinocoder) [inline docs/en-US/api/v1/endpoints/logs.md]: Delete

File: docs/en-US/api/v1/endpoints/logs.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+      "Meta": "admin,Admin"
+    },
+    ...
+  ],
+  "Sort": null,
+  "Extra": null
+}
+```
+
+# User Activity
```
Comment (pandinocoder) [inline docs/en-US/api/v1/endpoints/logs.md]: Should be `## User Activity`

File: docs/en-US/api/v1/endpoints/logs.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+      "Meta": null
+    },
+    ...
+  ],
+  "Sort": null,
+  "Extra": null
+}
+```
+
+# User activity by ip
```
Comment (pandinocoder) [inline docs/en-US/api/v1/endpoints/logs.md]: `## User Activity by IP`

File: docs/en-US/api/v1/endpoints/logs.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+
+## Authorized
```
Comment (pandinocoder) [inline docs/en-US/api/v1/endpoints/logs.md]: Delete

**Review by pandinocoder — CHANGES_REQUESTED** [review]

**Review by pandinocoder — CHANGES_REQUESTED** [review]
Will the new logs page be translated into French in this PR as well?

**Comment by Thomasfds** [comment]
I don't think so, I wouldn't have time for this right now

**Review by pandinocoder — APPROVED** [review]

**Pull request merged**